__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const { topicData, articleData, userData, commentData } = require('../db/data/test-data/index');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed({ topicData, userData, articleData, commentData });
```

Comment:
I would recommend having a look at the runSeed.js file for an alternative and shorter way to do this.